### DISCUSSION TITLE
Linux: permission denied

### DISCUSSION THREAD

**Discussion by dev-null-undefined** [EdgeTX/buddy#111] [Q&A] [answered]
Getting permision denied on linux

fixed by using chown on `/dev/bus/usb/003/014`
To be the "file" was owned by root so I hade to
`sudo chown $USER: /dev/bus/usb/003/014`
Which then made it work.

For others with this issue to find the path to the device us `lsusb`

Which will give you something like
`Bus XXX Device YYY: ID AAAA:BBBB ... SMT ... in DFU mode`

You can double check the AAAA:BBBB it should be same as on the website and then the path is `/dev/bus/usb/XXX/YYY`

It could be helpful to provide this information if possible in the error message when permission denied encountered on linux.

**Comment by pfeerick** [comment, marked as answer]
The 'normal' fix is to add yourself to the `dialout` group ... i.e. `sudo gpasswd --add ${USER} dialout` ... , and then log out/in so the group/permission change is applied. btw, the VID:PID will always be `0483:df11` for STM32 DFU mode USB device. 

But yes, a link to some troubleshooting steps would be a good idea.

**Comment by raphaelcoeffic** [comment]
The second way to solve this is to change the `udev` rules to give permission to everyone.
